freqtrade/tests/optimize/test_backtesting.py

```diff
@@ -139,18 +127,24 @@ def test_backtest2(default_conf, mocker):
     mocker.patch.dict('freqtrade.main._CONF', default_conf)
     data = optimize.load_data(ticker_interval=5, pairs=['BTC_ETH'])
     results = backtest(default_conf['stake_amount'], optimize.preprocess(data), 10, True)
-    num_resutls = len(results)
-    assert num_resutls > 0
+    num_results = len(results)
+    assert num_results > 0
```

Comment:
With lists, you can use `assert results` instead of `assert len(results) > 0` and with Pandas dataframes you can use `assert not results.empty`.